docs/usage.md

```diff
@@ -0,0 +1,213 @@
+
+Specifies the number of concurrent threads used for processing CSVs.
+
+You need multiple CSVs to take advantage of concurrent processing. TIP: In case your generated CSV is very large, you can use [the split_csv tool](https://github.com/datacommonsorg/data/tree/master/tools#csv-splitter) to shard it into multiple files.
+
+Defaults to `1`.
+
+### `-o`, `--output-dir=<outputDir>`
+
+Specifies the directory to write output files.
+
+Default is `dc_generated/` within current working directory.
+
+### `-s`, `--stat-checks`
+
+Checks integrity of time series by checking for holes, variance in values, etc.
+
+A set of counters detailing the results of the checks will be logged in `report.json`. For every such counter, the tool will provide a few exemplar cases to help the user
+understand and resolve the issue(s).
+
+For example, in this test input [`covid.mcf`](../tool/src/test/resources/org/datacommons/tool/lint/statchecks/input/covid.mcf) file, the value of the `CumulativeCount_MedicalTest_ConditionCOVID_19_Positive` StatVar for place
+`geoId/07` is `3.0` one day, (2020-03-02;line 49), and `7.0` on the next day (2020-03-03; line 65).
+Because the fluctuation in the value is greater than 100%, the tool flags this as a
+potential statistical issue. This is logged in the resulting [`report.json`](../tool/src/test/resources/org/datacommons/tool/lint/statchecks/output/report.json) as follows: 
+  ```json
+  "statsCheckSummary": [{
+      "placeDcid": "geoId/07",
+      "statVarDcid": "CumulativeCount_MedicalTest_ConditionCOVID_19_Positive",
+      "measurementMethod": "",
+      "observationPeriod": "",
+      "scalingFactor": "",
+      "unit": "",
+      "validationCounters": [{
+        "counterKey": "StatsCheck_MaxPercentFluctuationGreaterThan100",
+        "problemPoints": [{
+          "date": "2020-03-02",
+          "values": [{
+            "value": 3.0,
+            "locations": [{
+              "file": "covid.mcf",
+              "lineNumber": "49"
+            }]
+          }]
+        }, {
+          "date": "2020-03-03",
+          "values": [{
+            "value": 7.0,
+            "locations": [{
+              "file": "covid.mcf",
+              "lineNumber": "65"
+            }]
+          }]
+        }],
+        "percentDifference": 133.33
+      }]
+    }
+  ```
+Note  that information relevant to this check (sample place, file and location of the issue,
+the values involved, and the exact percent fluctuation) are conveniently provided
+to assist the user in debugging issues.
+
+
+Defaults to `true`.
+
+### `-p`, `--sample-places=<samplePlaces>`
+
+Specifies a list of place dcids to run stats check on.
+
+This flag should only be set if `--stat-checks` is `true`. If `--stat-checks` is `true` and this flag is not set, 5 sample places are picked for roughly each distinct place type.
+
+### `-r`, `--resolution=<resolutionMode>`
+
+Specifies the mode of resolution to use: `NONE`, `LOCAL`, or `FULL`.
```

Comment:
nit: This can be removed given the listing below?

Reply:
I'd prefer the keep it as a quick-reference for experienced users just looking for valid inputs